mmv1/products/cloudrun/api.yaml

```diff
@@ -18,9 +18,11 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://{{location}}-run.googleapis.com/
+    validator_base_url: https://run.googleapis.com/
   - !ruby/object:Api::Product::Version
     name: beta
     base_url: https://{{location}}-run.googleapis.com/
+    validator_base_url: https://run.googleapis.com/
```

Comment:
Theoretically, we don't need this for beta because we don't support beta resources - but also, it probably doesn't hurt to add it just in case.